CRM/Mailing/MailStore/Imap.php

```diff
@@ -54,7 +56,12 @@ public function __construct($host, $username, $password, $ssl = TRUE, $folder =
       'uidReferencing' => TRUE,
     ];
     $this->_transport = new ezcMailImapTransport($host, NULL, $options);
-    $this->_transport->authenticate($username, $password);
+    if ($userXOAUTH2) {
```

Comment:
This is a typo but I'm having a brain freeze at the moment as to why it's working as expected.

Reply:
It seems gmail, at least for now, will still let you use auth login, so inadvertently I've been testing regular logins, which in a way is good since it proves that still works with the changes. But the OAUTH login is NOT working, but I'm not sure yet if it's config in my gmail account or if the code changes aren't working.

Reply:
Fixed this now